version lib
don't redo all target for install
correct the install path

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -19,7 +19,7 @@ nls ?= 1
 xcb ?= 1
 
 bin = radeontop
-xcblib = libradeontop_xcb.so
+xcblib = libradeontop_xcb.so.${LIBradeontop_xcb_VERSION}
 src = $(filter-out amdgpu.c auth_xcb.c,$(wildcard *.c))
 verh = include/version.h
 
@@ -80,7 +80,6 @@ LIBS += $(shell pkg-config --libs libdrm)
 LIBS += -lm
 ifeq ($(xcb), 1)
 	xcb_LIBS += $(shell pkg-config --libs xcb xcb-dri2)
-	LIBS += -ldl
 endif
 
 # On some distros, you might have to change this to ncursesw
@@ -116,12 +115,12 @@ trans:
 	xgettext -o translations/radeontop.pot -k_ *.c \
 	--package-name radeontop
 
-install: all
-	$(INSTALL) -D -m755 $(bin) $(DESTDIR)/$(PREFIX)/bin/$(bin)
+install:
+	$(INSTALL) -D -m755 $(bin) $(PREFIX)/bin/$(bin)
 ifeq ($(xcb), 1)
-	$(INSTALL) -D -m755 $(xcblib) $(DESTDIR)/$(PREFIX)/$(LIBDIR)/$(xcblib)
+	$(INSTALL) -D -m755 $(xcblib) $(PREFIX)/$(LIBDIR)/$(xcblib)
 endif
-	$(INSTALL) -D -m644 radeontop.1 $(DESTDIR)/$(PREFIX)/$(MANDIR)/man1/radeontop.1
+	$(INSTALL) -D -m644 radeontop.1 $(PREFIX)/$(MANDIR)/man1/radeontop.1
 ifeq ($(nls), 1)
 	$(MAKE) -C translations install PREFIX=$(PREFIX)
 endif
